dive_core/command_hierarchy.h

```diff
@@ -445,6 +457,31 @@ class CommandHierarchyCreator : public EmulateCallbacksBase
                           uint64_t              va_addr,
                           Pm4Header             header) override;
 
+    void CreateTopologies();
+
+    virtual void OnSubmitStart(uint32_t submit_index, const SubmitInfo &submit_info) override;
+    virtual void OnSubmitEnd(uint32_t submit_index, const SubmitInfo &submit_info) override;
+
+    const DiveVector<DiveVector<uint64_t>> &get_node_children(uint64_t type, size_t sub_index) const
```

Comment:
the member func should follow the same naming convention as the other ones

Reply:
Done.